app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1578,11 +1581,19 @@ class BrowserTabFragment :
         }
 
         setFragmentResultListener(RESULT_KEY_CREDENTIAL_RESULT_SAVE) { _, result ->
-            autofillCredentialsSelectionResultHandler.processSaveCredentialsResult(result, viewModel)
+            launch {
+                autofillCredentialsSelectionResultHandler.processSaveCredentialsResult(result, viewModel)?.let {
+                    showAuthenticationSavedOrUpdatedSnackbar(it, R.string.autofillLoginSavedSnackbarMessage)
+                }
+            }
         }
 
         setFragmentResultListener(RESULT_KEY_CREDENTIAL_RESULT_UPDATE) { _, result ->
-            autofillCredentialsSelectionResultHandler.processUpdateCredentialsResult(result, viewModel)
+            launch {
+                autofillCredentialsSelectionResultHandler.processUpdateCredentialsResult(result, viewModel)?.let {
+                    showAuthenticationSavedOrUpdatedSnackbar(it, R.string.autofillLoginUpdatedSnackbarMessage)
```

Comment:
NIT: I really think we shouldn't be adding more logic into the BrowserTabFragment due to its size. Would be better to move out the call to  showAuthenticationSavedOrUpdatedSnackbar into autofillCredentialsSelectionResultHandler or just move it in its own class.

But could be looked into more later

Reply:
Will capture in here as a follow-up: [Autofill: Move more of autofill dialogs into autofill-impl module](https://app.asana.com/0/1202727816093912/1202727816093912/f)